### ISSUE TITLE
bug vaxImpact

### ISSUE THREAD

**Issue by ClementRolinat** [CovidTrackerFr/covidtracker-data#1]
Hello,

there is a bug in vaxImpact for the pcr+ graph titles: they do not correspond to the correct age class: 
[images/charts/france/pcr_plus_proportion_selon_statut_vaccinal_[20,39].jpeg](https://github.com/CovidTrackerFr/covidtracker-data/blob/67b559c33da5fe7ceac11928bd82c2fcaff86828/images/charts/france/pcr_plus_proportion_selon_statut_vaccinal_%5B20,39%5D.jpeg)
[images/charts/france/pcr_plus_proportion_selon_statut_vaccinal_[40,59].jpeg](https://github.com/CovidTrackerFr/covidtracker-data/blob/67b559c33da5fe7ceac11928bd82c2fcaff86828/images/charts/france/pcr_plus_proportion_selon_statut_vaccinal_%5B40,59%5D.jpeg)
[images/charts/france/pcr_plus_proportion_selon_statut_vaccinal_[60,79].jpeg](https://github.com/CovidTrackerFr/covidtracker-data/blob/67b559c33da5fe7ceac11928bd82c2fcaff86828/images/charts/france/pcr_plus_proportion_selon_statut_vaccinal_%5B60,79%5D.jpeg)
[images/charts/france/pcr_plus_proportion_selon_statut_vaccinal_[80;+].jpeg](https://github.com/CovidTrackerFr/covidtracker-data/blob/67b559c33da5fe7ceac11928bd82c2fcaff86828/images/charts/france/pcr_plus_proportion_selon_statut_vaccinal_%5B80;+%5D.jpeg)

also, the pcr+ graph for age class 0 to 19 is not updated anymore (15 days ago versus 8 days ago for the other graphs), so both bug may be linked.

**Comment by ClementRolinat** [comment]
@rozierguillaume @BrunoSpy @nicolinuxfr @NathanFallet  ? Can you check on this issue & my PR please ? It shouldn't take to long, my fix takes one line.

**Comment by ClementRolinat** [comment]
The bug is fixed, so i'm closing this issue.

**Comment by mikael-s** [comment]
Hi!

Well, I don't think the issue is fixed, those images still have wrong titles (and the legend could be improved as two different colors have the same title « Vaccinés »).

Thanks!

**Comment by ClementRolinat** [comment]
Actually, the linked images in the issue refer to a former commit where the bug was visible, thus it is expected that they show the bug. 
I closed the issue 4 days ago because at this time, the bug disappeared on the VaxImpact website (the graph title were correct, and the age class 0-19 was updated).
However, today the bug is once again visible on the VaxImpact website, thus I'm reopening the issue.
